community/installation-guides/daemon/centos8.md

```diff
@@ -0,0 +1,48 @@
+# CentOS 8
+In this guide we will install Pterodactyl's Daemon (wings) — including all of it's dependencies — and configure it to use a SSL connection.
+
+[[toc]]
+
+::: tip
+This guide is based off the [official installation documentation](/daemon/installing.md) but is tailored specifically for CentOS 8.
+:::
+
+## Install Requirements
+We will first begin by installing all of the Daemon's [required](/daemon/installing.md#dependencies) dependencies.
+
+### General Requirements
+```bash
+dnf install -y tar unzip make gcc gcc-c++ python2
+
+alternatives --set python /usr/bin/python2
```

Comment:
Why do we need python2? It's EOL soon, isn't it?

Reply:
Needed to build mmagic

Reply:
oh well